cluster/gce/gci/configure-helper-common.sh

```diff
@@ -2445,6 +2448,67 @@ function start-workload-controller-manager {
   cp "${src_file}" /etc/kubernetes/manifests
 }
 
+# Starts arktos-network controller.
+# It prepares the log file, loads the docker image, calculates variables, sets them
+# in the manifest file, and then copies the manifest file to /etc/kubernetes/manifests.
+#
+# Assumed vars (which are calculated in function compute-master-manifest-variables)
+#   CLOUD_CONFIG_OPT
+#   CLOUD_CONFIG_VOLUME
+#   CLOUD_CONFIG_MOUNT
+#   DOCKER_REGISTRY
+function start-arktos-network-controller {
+  local SALT=${1:-}
+  mkdir -p /etc/srv/kubernetes/arktos-network-controller
+  echo "Start arktos-network-controller"
+  local -r src_dir="${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty"
+  local -r dst_dir="/etc/srv/kubernetes/arktos-network-controller"
+  prepare-log-file /var/log/arktos-network-controller.log
+
+  cp "${src_dir}/crd-network.yaml" ${dst_dir}
+
+  # apply network crd before start network-controller
+  kubectl apply -f "${dst_dir}/crd-network.yaml"
+
+  # Calculate variables and assemble the command line.
+  local params="${ARKTOS_NETWORK_CONTROLLER_TEST_LOG_LEVEL:-"--v=4"}"
+  params+=" --master=http://127.0.0.1:8080"
+  params+=" --kube-apiserver-ip ${KUBERNETES_MASTER_INTERNAL_IP}"
+  if [[ "${USE_INSECURE_SCALEOUT_CLUSTER_MODE:-false}" == "true" ]]; then
```

Comment:
not sure whether we should use this env as controlling condition.

Reply:
for now, arktos is using USE_INSECURE_SCALEOUT_CLUSTER_MODE for insecure mode. but default, we are using  secure port: 443 to access